.github/workflows/build-and-upload-staging.yml

```diff
@@ -2,69 +2,26 @@
 # SPDX-License-Identifier: Apache-2.0
 
 name: Build And Upload Staging Artifact
```

Comment:
name of this workflow will be misleading since we are de-coupling this with our release. Maybe call it "Build and Upload Integ Test Artifact"

Reply:
Done